framework-plugins/lisk-framework-http-api-plugin/src/controllers/transactions.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { isBase64String, validator, LiskValidationError } from '@liskhq/lisk-validator';
+import { Request, Response } from 'express';
+import { BaseChannel, PluginCodec } from 'lisk-framework';
+
+const transactionInputSchema = {
```

Comment:
This schema is not available elsewhere in the code to be imported?

Reply:
Its schema for API endpoint parameters, not for the transaction itself. We can re-use it from `swagger.yml` but as discussed earlier swagger will only be used for documentation, so I declared the schema here.